app/views/categories/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<% provide(:title, "Categories") %>
+<p class='user'><%= "Welcome #{current_user.fullname}" %></p>
+
+<div>
+  <% @categories.each do |category| %>
+    <%= render category %>
+  <% end %>
+</div>
+
+<button class='add-category btn flex'>
+<div class='new-category btn flex'>
+  <%= link_to "Add a new Category", new_category_path %>
+  <div>
```

Comment:
- [ ] Kindly note that the `add new categories` and `new transaction` buttons are always displayed at the bottom of the view-heigh even though the list of all categories/transactions is empty. Check the deference here :point_down: 

your design | the original template
 -------|----------
![image](https://user-images.githubusercontent.com/65068771/170594660-b72dc7e1-e52f-4307-8c79-20a56c15c716.png) | ![image](https://user-images.githubusercontent.com/65068771/170594851-2ab0dff1-53d8-4b75-9842-40b9f607025e.png)

- [ ] Kindly also remove the margin around your app as the original template does not have one.
- [ ] Make sure also you are using the same `font-family`

Reply:
Done 👍 thank you for pointing these out 😄